src/TimeSeries/widget/components/NVD3LineChart.ts

```diff
@@ -0,0 +1,99 @@
+import { Selection, select } from "d3";
+import "nvd3";
+import { Component, DOM } from "react";
+
+import { Series } from "./TimeSeries";
+
+export interface Nvd3LineChartProps {
+    height?: number;
+    width?: number;
+    chartProps?: any;
+    datum: Series[];
+}
+
+export function isPlainObject(object: any): boolean {
```

Comment:
For unit  testing, and to improve the test code coverage.
Should we make it private, or make it static or move it to a utils module?